platforms/corda-ent/configuration/roles/helm_component/templates/create_signer.tpl

```diff
@@ -17,42 +17,30 @@ spec:
       namespace: {{ component_ns }}
     replicas: 1
     image:
+      initContainerName: {{ network.docker.url }}/alpine-utils:1.0
```

Comment:
corresponding changes as values needed after the comments are addressed.

Reply:
Fixed in [ae9a768](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/992/commits/ae9a76899ddcce9ea9e2067c3ba79ebc784c08c5); changes to `value.yaml` file have been worked into `create_signer.tpl`